app/models/support_interface/provider_user_form.rb

```diff
@@ -34,5 +35,15 @@ def self.from_provider_user(provider_user)
         provider_ids: provider_user.provider_ids,
       )
     end
+
+  private
+
+    def email_is_unique
```

Comment:
Do we need to downcase `email_address` here (or earlier) to ensure that we can't duplicates that differ only by case?

Reply:
Great point - fixed in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1124/commits/416a5772dbb765558cdbf45432ef7a37b7cb2e54.